subprojects/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/test/TestTaskPropertiesServiceIntegrationTest.groovy

```diff
@@ -27,10 +27,10 @@ import org.gradle.test.fixtures.plugin.PluginBuilder
 
 import java.util.stream.Stream
 
+@UnsupportedWithConfigurationCache(because = "uses CC-incompatible APIs")
```

Comment:
:question: What is the incompatible API being used here?

Reply:
It was mainly due to accessing `Task.getProject()` during execution. I've now changed the test code to make it compatible.